zds/utils/models.py

```diff
@@ -217,32 +217,17 @@ class Meta:
         verbose_name_plural = 'Alertes'
 
 
-class CommentLike(models.Model):
+class CommentVote(models.Model):
 
-    """Set of like comments."""
+    """Set of comment votes."""
     class Meta:
-        verbose_name = 'Ce message est utile'
-        verbose_name_plural = 'Ces messages sont utiles'
+        verbose_name = 'Vote'
+        verbose_name_plural = 'Votes'
+        unique_together = ('user', 'comment')
```

Comment:
ou simplement faire un try/catch sur les +/- foireux

Le 28/03/2016 21:52, Hugo Courtecuisse a écrit :
>
> In zds/utils/models.py 
> <https://github.com/zestedesavoir/zds-site/pull/3265#discussion_r57622736>:
>
> >      class Meta:
> > -        verbose_name = 'Ce message est utile'
> > -        verbose_name_plural = 'Ces messages sont utiles'
> > +        verbose_name = 'Vote'
> > +        verbose_name_plural = 'Votes'
> > +        unique_together = ('user', 'comment')
>
> En prod, y'a des gens qui ont voté plusieurs fois sur le même message 
> (plusieurs plus et moins) dont moi. Du coup, il faudrait en tenir 
> compte quand le script de migration va passer. Si vous mettez un 
> unique le script de migration qui reprend l'existant à de grande 
> chance de casser.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3265/files/d76c0d309b5aef90d8249c772121840cd9bf4e31#r57622736>
>